constants.py

```diff
@@ -1,11 +1,28 @@
-# constants that used in javascript and python
-MINIMAL_ZOOM = 16
+class CONST(object):
```

Comment:
Classes are in camel case, not all caps (should be "Const" or better "Constants").